src/css/Menu.css

```diff
@@ -8,6 +8,21 @@
     align-items: center;
 }
 
+.navegacao-menu {
+    width: 100%;
+    max-width: 700px;
+    margin: 0 15px 0 15px;
```

Comment:
Aqui dá pra deixar como `margin: 0 15px` se realmente for necessário zerar essas medidas.